pyproject.toml

```diff
@@ -236,6 +237,8 @@ ignore = [
   "INP001",
   # Errors should end with "Error"
   "N818",
+  # mypy prevents blanket-type-ignore
+  "PGH003",
```

Comment:
Oh interesting. I was wondering why this hadn’t fired on the ruff side of things. Thought I had seen some before that weren’t ruff ignited but maybe not. So I guess we could have left the mypy config as is and removed the ruff ignores 🤷 but either way this works.